Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -245,6 +259,19 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
             'HumanReadableFormat': formats['markdown'],
         }
         demisto.results(entry)
+    demisto.results(set([x[DBOT_TAG_FIELD] for x in data]))
+    if len(set([x[DBOT_TAG_FIELD] for x in data])) == 1:
+        single_label = [x[DBOT_TAG_FIELD] for x in data][0]
+        err = ['All received incidents have the same label: {}.'.format(single_label)]
+        err += ['At least 2 different labels are required to train a classifier.']
+        if labels_mapping == ALL_LABELS:
+            err += ['Please make sure that incidents of at least 2 labels exist in the environment.']
+        else:
+            unfound_mapped_label = [l for l in labels_mapping if l not in exist_labels_counter
```

Comment:
This message is for label that the user define but does not exist in the data probably a mistake of the user